### ISSUE TITLE
How would I go about using it on termux??

### ISSUE THREAD

**Issue by prestige2** [jhead/phantom#134]
I know servers are out on PS4 now but u cant add Ur own servers meaning this is still useful. How would I go about using this on Termux?? Itd be cool being able to use this when going to friends houses via just Ur phone!

**Comment by jhead** [comment]
Hey @astrolul! Sorry about the delay. Have you checked out the comments here? https://github.com/jhead/phantom/issues/111#issuecomment-686714910

I believe some folks have been able to get phantom working on Termux, but sadly I don't have an Android device to confirm or write up my own instructions.

**Comment by astrolul** [comment]
> Hey @astrolul! Sorry about the delay. Have you checked out the comments here? https://github.com/jhead/phantom/issues/111#issuecomment-686714910
> 
> I believe some folks have been able to get phantom working on Termux, but sadly I don't have an Android device to confirm or write up my own instructions.

Yeah I did but it would be good if someone could give us a tutorial on it. Since termux is a Linux based thing how do I run it in normal Linux then??

**Comment by azariah001** [comment]
In the linked issue BadWolf22 provides a pretty solid guide on the setup. https://github.com/BadWolf22/JavaRock

It looks simply to be a matter of pulling the latest release executable for arm5 in termux, making it executable, and running it.
